include/ipr/interface

```diff
@@ -1366,6 +1389,15 @@ namespace ipr {
       Arg2_type args() const { return second(); }
    };
 
+                                // -- Coerce --
+   // A generalized type conversion either built-in to the language or
+   // programmatically defined by either constructors or conversion
+   // functions.
+   struct Coerce : Binary<Category<conditional_cat, Classic>, const Type&> {
+      Arg1_type target() const { return first(); }
+      Arg2_type expr() const { return second(); }
+   };
```

Comment:
I was expecting `target()` to be the second operand and `source()` (instead of `expr()`) to be first operand